docs/openapi/openapi.yaml

```diff
@@ -219,16 +219,73 @@ components:
     InitialLpa:
       type: object
       required:
+        - lpaType
         - donor
         - attorneys
+        - certificateProvider
+        - signedAt
       properties:
+        lpaType:
+          type: string
+          enum:
+            - property-and-affairs
+            - personal-welfare
         donor:
           $ref: "#/components/schemas/Donor"
         attorneys:
           type: array
           items:
             $ref: "#/components/schemas/Attorney"
           minLength: 1
+        trustCorporations:
+          type: array
+          items:
+            $ref: "#/components/schemas/TrustCorporation"
```

Comment:
Are trust corporations just a different kind of attorney? Wondering if we can mix them into `attorneys` rather than create a separate array, since that's historically been a bit of a pain.

Reply:
Though possibly less of an issue now that we don't also have a `replacementAttorneys` array TBH

Reply:
We definitely could do that, although having them separate does make the de/serialisation simpler. Another thing I'm wondering is whether it would be better to have `activeAttorneys`/`replacementAttorneys` instead of status?

Reply:
Whereas separate arrays make sense during application, the active/replacement merge was intentional to make attorneys easier to manage post-registration. We had to add an "active" flag in Sirius to note ex-attorneys, which effectively means using two control points to create a status field. The merged approach gives us a bit more flexibility to manage these statuses.

Fair point about de/serialisation of attorneys vs TCs, having mixed structures would open up other problems. Happy to see how it goes like this and we can always revisit (either of these things) later.